src/ContainerHeader.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import cockpit from 'cockpit';
+import Select from '../lib/cockpit-components-select.jsx';
+
+const _ = cockpit.gettext;cockpit.gett
+
+class ContainerHeader extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            filter: 'running',
+            filterText: ''
+        };
+
+        this.filterChanged = this.filterChanged.bind(this);
```

Comment:
This doesn't seem necessary, you only call this from a method that already has a correct `this`.